plugins/woocommerce/client/admin/client/settings-payments/components/other-payment-gateways/other-payment-gateways.tsx

```diff
@@ -61,72 +86,112 @@ export const OtherPaymentGateways = ( {
 				<GridItemPlaceholder />
 			</>
 		) : (
-			suggestions.map( ( extension ) => (
-				<div
-					className="other-payment-gateways__content__grid-item"
-					key={ extension.id }
-				>
-					<img src={ extension.icon } alt={ extension.title } />
-					<div className="other-payment-gateways__content__grid-item__content">
-						<span className="other-payment-gateways__content__grid-item__content__title">
-							{ extension.title }
-						</span>
-						<span className="other-payment-gateways__content__grid-item__content__description">
-							{ extension.description }
-						</span>
-						<div className="other-payment-gateways__content__grid-item__content__actions">
-							<Button
-								variant="primary"
-								onClick={ () =>
-									setupPlugin(
-										extension.id,
-										extension.plugin.slug
-									)
-								}
-								isBusy={ installingPlugin === extension.id }
-								disabled={ !! installingPlugin }
-							>
-								{ __( 'Install', 'woocommerce' ) }
-							</Button>
+			suggestionsByCategory.map(
+				( { category, suggestions: categorySuggestions } ) => {
+					if ( categorySuggestions.length === 0 ) {
+						return null;
+					}
+
+					return (
+						<div key={ category.id }>
+							<div className="other-payment-gateways__content__title">
+								<h3 className="other-payment-gateways__content__title__h3">
+									{ category.title }
+								</h3>
+								<Tooltip
+									text={ category.description }
+									position="top right"
+								>
+									<Gridicon
+										icon="info-outline"
+										className="other-payment-gateways__content__title__tooltip"
+									/>
+								</Tooltip>
+							</div>
+
+							<div className="other-payment-gateways__content__grid">
+								{ categorySuggestions.map( ( extension ) => (
+									<div
```

Comment:
We could consider creating a separate component for a Suggested Extension, but since we're only using it here right now, it's probably overkill.